src/components/templates/Jazzbar/components/JazzBarTableComponent/JazzBarTableComponent.scss

```diff
@@ -0,0 +1,129 @@
+@import "scss/constants.scss";
```

Comment:
This file seems to duplicate our generic components styles from `src/components/molecules/TableComponent/TableComponent.scss`

What's the reasoning/need for duplicating it?

Reply:
It seems the only differences here are:

```diff
diff --git a/src/components/molecules/TableComponent/TableComponent.scss b/src/components/templates/Jazzbar/components/JazzBarTableComponent/JazzBarTableComponent.scss
index d2bf2f8d..d0532c3b 100644
--- a/src/components/molecules/TableComponent/TableComponent.scss
+++ b/src/components/templates/Jazzbar/components/JazzBarTableComponent/JazzBarTableComponent.scss
@@ -1,40 +1,40 @@
 @import "scss/constants.scss";
 
-.table-component-container {
+.jazzbar-table-component-container {
   display: flex;
   justify-content: center;
   flex-direction: column;
   align-items: center;
 
-  &.jazzbar-table-1 {
-    grid-column: 1;
-    grid-row: 1;
-  }
+  // &.jazzbar-table-1 {
+  //   grid-column: 1;
+  //   grid-row: 2;
+  // }
 
-  &.jazzbar-table-2 {
-    grid-column: 7;
-    grid-row: 1;
-  }
+  // &.jazzbar-table-2 {
+  //   grid-column: 7;
+  //   grid-row: 1;
+  // }
 
-  &.jazzbar-table-3 {
-    grid-column: 1;
-    grid-row: 2;
-  }
+  // &.jazzbar-table-3 {
+  //   grid-column: 1;
+  //   grid-row: 2;
+  // }
 
-  &.jazzbar-table-4 {
-    grid-column: 7;
-    grid-row: 2;
-  }
+  // &.jazzbar-table-4 {
+  //   grid-column: 7;
+  //   grid-row: 2;
+  // }
 
-  &.jazzbar-table-5 {
-    grid-column: 1;
-    grid-row: 3;
-  }
+  // &.jazzbar-table-5 {
+  //   grid-column: 1;
+  //   grid-row: 3;
+  // }
 
-  &.jazzbar-table-6 {
-    grid-column: 7;
-    grid-row: 3;
-  }
+  // &.jazzbar-table-6 {
+  //   grid-column: 7;
+  //   grid-row: 3;
+  // }
 
   .table-title-container {
     margin: 20px 0;
@@ -55,16 +55,17 @@
   }
 
   .table-item {
+    height: 100%;
+    width: 100%;
     background: $dark;
     border-radius: 22px;
-    min-height: 30px;
+
     color: $white;
     display: flex;
     justify-content: space-around;
     flex-wrap: wrap;
     padding: 40px 25px 10px 25px;
     position: relative;
-    margin: 5px 5px;
     &:hover {
       background-color: lighten($dark, 10%);
       transform: scale(1.05);
```